Index: sblim-sfcb-1.4.9/mofc/Makefile.am
===================================================================
--- sblim-sfcb-1.4.9.orig/mofc/Makefile.am
+++ sblim-sfcb-1.4.9/mofc/Makefile.am
@@ -21,7 +21,7 @@ endif
 mofc_SOURCES=mofc.y mofs.l mofparse.c hash.h hash.c symtab.h symtab.c mofdecl.h backend.h 
 EXTRA_mofc_SOURCES=backend_print.c backend_sfcb.c
 mofc_DEPENDENCIES=mofc-@BACKEND_OBJ@
-mofc_LDADD=mofc-@BACKEND_OBJ@ @BACKEND_LIBS@ @LEXLIB@
+mofc_LDADD=mofc-@BACKEND_OBJ@ @BACKEND_LIBS@ @LEXLIB@ -ldl
 mofc_CPPFLAGS=@BACKEND_INCLUDES@
 
 EXTRA_DIST=runtest.sh tests/test1.mof tests/include/test2.mof \
